cargo-shuttle/src/init.rs

```diff
@@ -2,47 +2,49 @@ use std::fs::{read_to_string, File};
 use std::io::Write;
 use std::path::{Path, PathBuf};
 
-use anyhow::Result;
+use anyhow::{bail, Result};
 use cargo_edit::{find, get_latest_dependency, registry_url};
 use indoc::indoc;
+use shuttle_common::project::ProjectName;
 use toml_edit::{value, Array, Document, Table};
 use url::Url;
 
 #[derive(Clone, Copy, Debug, PartialEq, Eq, strum::Display, strum::EnumIter)]
 #[strum(serialize_all = "kebab-case")]
```

Comment:
Are these still used? Or should they be moved to the arg's enum?

Reply:
I agree that the enums are very likely redundant, but they will be reworked when git cloning is introduced, so IMO they can just be left like this until then.

Reply:
It looks like the derives are still used, yeah.